server/routers/router-rides.js

```diff
@@ -94,15 +94,47 @@ exports.getAllRiders = (req, res) => {
     });
 };
 
+const getRiders = car =>
+  knex.from('users')
+    .innerJoin('cars_users', 'users.id', 'cars_users.user_id')
+    .where('car_id', '=', car.id)
+    .then((riders) => {
+      car.riders = riders;
+      return car;
+    })
+    .catch((err) => {
+      console.log('error in getRiders.  will return empty array.  error is: ', err);
+      car.riders = [];
+      return car;
+    });
+
 exports.getAllRides = (req, res) => {
   const coveyId = req.params.coveyId;
 
   knex.from('cars')
-    .innerJoin('cars_users', 'cars.id', 'cars_users.car_id')
-    .innerJoin('users', 'users.id', 'cars_users.user_id')
+    // .innerJoin('cars_users', 'cars.id', 'cars_users.car_id')
+    // .innerJoin('users', 'users.id', 'cars_users.user_id')
     .where('covey_id', '=', coveyId)
     .then((cars) => {
-      res.status(200).json(cars);
+      const files = [];
```

Comment:
In the spirit of self-documenting code, can we change the variable name 'files' to something more intuitive? E.g., resources or resourcesArray.